config/seeds.yml

```diff
@@ -157,35 +235,66 @@ permissions:
  - { role: trader, verb: all, path: api/v2/barong/resource, action: accept }
  # ranger access
  - { role: trader, verb: get, path: api/v2/ranger, action: accept }
- #applogic access
+ # Finex upstream access
+ - { role: trader, verb: get, path: api/v2/open_finance, action: accept }
+ # applogic access
  - { role: trader, verb: all, path: api/v2/applogic, action: accept }
 
- # MEMBER
+ # BROKER
  # has standard user permissions to ACCOUNT, MARKET, RESOURCE modules
+ - { role: broker, verb: all, path: api/v2/peatio/account, action: accept }
+ - { role: broker, verb: all, path: api/v2/peatio/market, action: accept }
+ - { role: broker, verb: all, path: api/v2/finex/market, action: accept }
+ - { role: broker, verb: all, path: api/v2/barong/resource, action: accept }
+ # ranger access
+ - { role: broker, verb: get, path: api/v2/ranger, action: accept }
+ # Finex upstream access
+ - { role: broker, verb: get, path: api/v2/open_finance, action: accept }
+ #applogic access
+ - { role: broker, verb: all, path: api/v2/applogic, action: accept }
+
+ # MEMBER
  - { role: member, verb: all, path: api/v2/peatio/account, action: accept }
  - { role: member, verb: all, path: api/v2/peatio/market, action: accept }
  - { role: member, verb: all, path: api/v2/finex/market, action: accept }
  - { role: member, verb: all, path: api/v2/barong/resource, action: accept }
  # ranger access
  - { role: member, verb: get, path: api/v2/ranger, action: accept }
+ # Finex upstream access
+ - { role: member, verb: get, path: api/v2/open_finance, action: accept }
  # applogic access
  - { role: member, verb: all, path: api/v2/applogic, action: accept }
 
- # Manager
- # has standard user permissions to ACCOUNT, MARKET, RESOURCE modules
- - { role: manager, verb: all, path: api/v2/peatio/account, action: accept }
- - { role: manager, verb: all, path: api/v2/peatio/market, action: accept }
- - { role: manager, verb: all, path: api/v2/barong/resource, action: accept }
+ # MAKER
+ - { role: maker, verb: all, path: api/v2/peatio/account, action: accept }
+ - { role: maker, verb: all, path: api/v2/peatio/market, action: accept }
+ - { role: maker, verb: all, path: api/v2/finex/market, action: accept }
+ - { role: maker, verb: all, path: api/v2/barong/resource, action: accept }
  # ranger access
- - { role: manager, verb: get, path: api/v2/ranger, action: accept }
+ - { role: maker, verb: get, path: api/v2/ranger, action: accept }
+ # Finex upstream access
+ - { role: maker, verb: get, path: api/v2/open_finance, action: accept }
  # applogic access
- - { role: manager, verb: all, path: api/v2/applogic, action: accept }
+ - { role: maker, verb: all, path: api/v2/applogic, action: accept }
 
  # Service Account
  # has standard user permissions to ACCOUNT, MARKET, RESOURCE modules
  - { role: service_account, verb: all, path: api/v2/peatio/account, action: accept }
```

Comment:
what is a service_account role??

Reply:
we can keep this one as default role to be used for market_making from a 3rd party. but renaming it sa_maker